cfe_internal/enterprise/federation/federation.cf

```diff
@@ -332,11 +332,15 @@ bundle agent clean_when_off
   classes:
     selinux_enabled.default:_stdlib_path_exists_semanage::
       "has_cftransport_fcontext" expression => returnszero("$(paths.semanage) fcontext -l | grep $(home)", "useshell");
+    "remote_hubs_table_empty" expression => returnszero(`[ $(const.dollar)($(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM remote_hubs") -eq "0"] && true || false`);
```

Comment:
The `&& true || false` part is not necessary, AFAICT.

Reply:
was an artifact of my doing echos there instead of true false. :+1: